frontend/src/views/host/process/process/index.vue

```diff
@@ -54,11 +54,17 @@
                     <el-table-column
                         :label="$t('process.memory')"
                         fix
+                        min-width="120"
                         prop="rssValue"
                         :formatter="memFormatter"
                         sortable
                     ></el-table-column>
-                    <el-table-column :label="$t('process.numConnections')" fix prop="numConnections"></el-table-column>
+                    <el-table-column
+                        :label="$t('process.numConnections')"
+                        fix
+                        prop="numConnections"
+                        min-width="120"
+                    ></el-table-column>
                     <el-table-column
                         :label="$t('process.status')"
                         fix
```

Comment:
There are no significant code differences in this snippet, but here are some minor suggestions:

1. The `min-width` attribute is set to `120px` for both columns with "Memory" and "Number of Connections". This should ensure that these columns have a minimum width of 120 pixels, which can improve readability on smaller screens.

Overall, the changes between lines 54 and 63 are minimal and do not introduce any potential issues.